src/jdk.jdwp.agent/share/native/libjdwp/eventFilter.c

```diff
@@ -1265,7 +1265,6 @@ enableEvents(HandlerNode *node)
         case EI_THREAD_END:
         case EI_VM_INIT:
         case EI_VM_DEATH:
-        case EI_CLASS_PREPARE:
```

Comment:
Since CLASS_PREPARE is no longer globally and permanently enabled, we remove it from this list so the code below can enable it as needed based on whether or not the debugger as requested CLASS_PREPARE events.